gap/NautyGraph.gi

```diff
@@ -361,6 +361,40 @@ InstallMethod( CanonicalForm,
     
 end );
 
+InstallMethod( CanonicalForm,
+               [ IsNautyEdgeColoredGraphRep ],
+               
+  function( graph )
+    local colors, edges, perm, new_graph, permEdges, i,
+        permInv, edgeClass, newClass;
+    
+    perm := CanonicalLabeling( graph );
```

Comment:
BTW is `perm` even used here?

Reply:
Good catch. The same is true for the other Canonical-method. I removed both.